src/main/java/at/hannibal2/skyhanni/features/misc/WhereWillIBe.kt

```diff
@@ -0,0 +1,46 @@
+package at.hannibal2.skyhanni.features.misc
+
+import at.hannibal2.skyhanni.data.IslandType
+import at.hannibal2.skyhanni.events.MessageSendToServerEvent
+import at.hannibal2.skyhanni.utils.ChatUtils
+import at.hannibal2.skyhanni.utils.LorenzUtils
+import at.hannibal2.skyhanni.utils.SimpleTimeMark
+import at.hannibal2.skyhanni.utils.SkyBlockTime
+import net.minecraftforge.fml.common.eventhandler.SubscribeEvent
+import java.text.SimpleDateFormat
+import java.util.Date
+import kotlin.math.pow
+import kotlin.random.Random
+
+object WhereWillIBe {
+    private val nonIslands = listOf(IslandType.NONE, IslandType.UNKNOWN, IslandType.MINESHAFT) // IslandType.MINESHAFT == pain
+    private val privateOrGuest = listOf(IslandType.PRIVATE_ISLAND_GUEST, IslandType.PRIVATE_ISLAND, IslandType.GARDEN_GUEST)
+    private val onceOnlyIslands = listOf(IslandType.DARK_AUCTION, IslandType.KUUDRA_ARENA, IslandType.CATACOMBS, IslandType.THE_RIFT) // add/remove islands as you see fit, but DARK_AUCTION should stay in here
+    private val islandsAsList = IslandType.entries.toList().filter { it !in nonIslands }.toMutableList()
+    @SubscribeEvent
+    fun onMessageSendToServer(event: MessageSendToServerEvent) {
+        if (!LorenzUtils.onHypixel) return
+        if (event.message.lowercase() != "/wherewillibe") return
+        event.setCanceled(true)
+        if (SkyBlockTime.now().month != 12) islandsAsList.remove(IslandType.WINTER)
```

Comment:
this will remove it from the list until the game restarts

Reply:
resolved